tensorflow-base-cpu-gpu-linwin/meta.yaml

```diff
@@ -118,18 +117,19 @@ requirements:
     - numpy >=1.20,<2
     - scipy >=1.7.3,<2
     - packaging
-    - absl-py >=0.10.0
+    - absl-py >=1.0.0
     - astunparse >=1.6.3
-    - flatbuffers >=1.12
+    - flatbuffers >=2.0
     - gast >=0.2.1
```

Comment:
Can `gast` above 0.4.0  break TF's tests? See https://github.com/tensorflow/tensorflow/blob/359c3cdfc5fabac82b3c70b3b6de2b0a8c16874f/tensorflow/tools/pip_package/setup.py#L80-L81

Reply:
Pinned to 0.4.0 as that is the version we want (we already explicitly pin it in the `osx-64` recipe).